pkgs/games/rpg-cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rpg-cli";
+  version = "0.2.0";
```

Comment:
```suggestion
  version = "unstable-2021-05-27";
```

Reply:
This shouldn't be unstable, since there have been releases already